src/features/ModelPlan/ModelToOperations/_components/AddCustomSolutionForm/index.tsx

```diff
@@ -116,7 +121,7 @@ const SolutionForm = ({ closeModal }: { closeModal: () => void }) => {
 
   return (
     <FormProvider {...methods}>
-      {message}
+      {errorMessageInModal}
```

Comment:
Instead of placing the errors in each of the forms, can we place it in the parent component once?  - `MTOModal`